be/src/exec/vectorized/sorting/compare_column.cpp

```diff
@@ -227,17 +232,27 @@ class ColumnTieBuilder final : public ColumnVisitorAdapter<ColumnTieBuilder> {
     template <typename T>
     Status do_visit(const vectorized::BinaryColumnBase<T>& column) {
         auto& data = column.get_data();
-        for (size_t i = 1; i < column.size(); i++) {
-            (*_tie)[i] &= SorterComparator<Slice>::compare(data[i - 1], data[i]) == 0;
+        TieIterator iterator(*_tie);
```

Comment:
Why change it ? Any performance benefit or regression ?

Reply:
reverted